VisitorTests/CodeGenerator/CodeGeneratorTest.cs

```diff
@@ -0,0 +1,125 @@
+﻿using GOAT_Compiler;
+using GOATCode.node;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VisitorTests.Utilities;
+using Xunit;
+
+namespace VisitorTests
+{
+    public class CodeGeneratorTest
+    {
+        [SkippableTheory(typeof(TestDependencyException))]
+        [InlineData("RelMoveTest.txt", "RelMoveTest.gcode")]
+        public void CreatedFileChecker(string file, string outPutFileName)
+        {
+            string TestFilePath = FileReadingTestUtilities.ProjectBaseDirectory + "CodeGenerator/Tests/InputFolder/"+ file;
+            Start s = FileReadingTestUtilities.ParseFile(TestFilePath);
+            ISymbolTable symbolTable = FileReadingTestUtilities.BuildSymbolTable(s);
+            TypeChecker typeChecker = FileReadingTestUtilities.DoTypeChecking(s, symbolTable);
+            
+            CodeGenerator codeGenerator;
```

Comment:
Dont declare codeGenerator here, just add the type to the line where it's used

Reply:
fixed